### PR TITLE
chore: added raw sequencing file Python script (#5)

### PR DESCRIPTION
**Pull request by jpaten**
Wrote a script to:
- Download the appropriate files from Github repositories
- Join and concatenate the files as instructed in (#5)
- Save the output file in `/files/out`
 checked to confirm that the concatenation worked properly, I would have added some sort of automatic check but it did not seem possible due to the simple nature of this script.

This script requires that the user has Python and Numpy installed in their active environment.

### CODE REVIEW

File: files/build-raw-sequencing-files.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+from pathlib import Path
+from urllib.parse import urlparse
+
+STORAGE_FOLDER_PATH = "./files/unprocessed_files/"
+OUTPUT_PATH = "./files/out/raw_sequencing_data.tsv"
+DEEPCONSENSUS_URL = "https://raw.githubusercontent.com/human-pangenomics/HPRC_metadata/main/data/sample-files/hprc_metadata_sample_files_DEEPCONSENSUS.tsv"
+HIFI_URL = "https://raw.githubusercontent.com/human-pangenomics/HPRC_metadata/main/data/sample-files/hprc_metadata_sample_files_HiFi.tsv"
+ONT_URL = "https://raw.githubusercontent.com/human-pangenomics/HPRC_metadata/main/data/sample-files/hprc_metadata_sample_files_ONT.tsv"
+METADATA_URLS = [DEEPCONSENSUS_URL, HIFI_URL, ONT_URL]
+TABLE_URL = "https://raw.githubusercontent.com/human-pangenomics/HPRC_metadata/main/data/production/hprc-production-biosample-table.tsv"
```
Comment (NoopDog) [inline files/build-raw-sequencing-files.py:13]: Can we call this "BIOSAMPLE_TABLE_URL" or similar to indicate the biosamples are in here?

**Comment by NoopDog** [comment]
Hi @jpaten thanks! Can we add a requirements.txt here? Thx!

**Review by NoopDog — CHANGES_REQUESTED** [review]
Can we add instructions on how to setup to run and how to run the script? Thx!

**Review by NoopDog — CHANGES_REQUESTED** [review]
Hi @jpaten, can you add a section to the project readme explaining how to set up to run this and how to run it?

**Review by hunterckx — CHANGES_REQUESTED** [review]
One request from me -- could you change the output path to `./files/source/raw-sequencing-data.tsv` rather than `./files/out/raw_sequencing_data.tsv`, so that it'll be directly available to the `build-catalog` script? Otherwise, looks good!

**Comment by jpaten** [comment]
> One request from me -- could you change the output path to `./files/source/raw-sequencing-data.tsv` rather than `./files/out/raw_sequencing_data.tsv`, so that it'll be directly available to the `build-catalog` script? Otherwise, looks good!

Fixed! Sorry, I didn't realize that was necessary, will remember it for #11 !

**Review by hunterckx — APPROVED** [review]
Thanks @jpaten!

**Pull request merged**